index.html

```diff
@@ -1,19 +1,45 @@
-Saulo Alberto Hernandez Ramos
+<!DOCTYPE html>
 
-About
+<html>
 
-    I am 26 year old expecting father. I was happily married at 21 years old. I currently work as a blind installer for Budget Blinds out of Durham, NC. I started here at CTD with the wishes of breaking into the IT world and forging a better life for myself and my family. I was born and raised in Juarez, Chihuahua, Mexico and moved to Garner, NC when I was 6 years old. After marrying my wife we moved to Raleigh for a few years and now live in Morrisville, NC. 
+<head>
+    <title> CTD Project </title>
+    <meta charshet="UTF-8">
+</head>
 
-Experience
+<body>
 
-    I have zero to little coding experience and it actually all was with CTD. When I was 16 years old I ran into this company through a non-profit organization I often volunteer for 'El Pueblo.' I have worked in almost every blue collar job you can think of as well as front end and back end sales. I was a service industry worker for 7 years up until the pandemic hit which I then transitioned to delivering pizzas for Papa Johns. After that got old and boring I went to work for Amazon out of Durham and eventually ended up where I am now at Budget Blinds. 
+<h1> 
+    <p>Saulo Alberto Hernandez Ramos</p>
```

Comment:
Usually no `<p>` inside `<h1>`: An `h` tag is a header, so it will almost always just be one line, and wouldn't be broken into paragraphs.